src/runtime/slot-proxy.ts

```diff
@@ -67,4 +67,16 @@ export class SlotProxy {
       handler(event);
     }
   }
+
+  /**
+   * gives all rendering capabilities from the current particle to the given one
```

Comment:
Two notes:
- This should be syntactically a full sentence (capitalized, ending punctuation) for easier readability.
- Please expand on what this means. Logic appears to show this is not just mutating the specified passed-in particle, it's also detaching/removing the old particle. Does the old particle lose its capabilities? That's not clear from the comment.

Reply:
Changed the comment! :D